### PR TITLE
Instruction Mining: When Data Mining Meets Large Language Model Finetuning

### PR DESCRIPTION
**Pull request by kcoost**
#34

### CODE REVIEW

File: papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Benchmarks
+
+- MT-bench loss
+- OpenLLM benchmarks
+- GPT-as-a-judge (not sure what dataset they used for this)
+
+## Metric Results
+
+There is a significant improvement over random subset selection at a relatively small cost. They show
```
Comment (Dahoas) [inline papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md:39]: Possibly some double descent phenomena: as you increase number of training samples.

Comment (Dahoas) [inline papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md:39]: Testing is done relatively in-distribution.

File: papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ID: 34
+
+## Summary
+
+This paper seeks to find an optimal subset to instruction-tune an LLM with. They do this by considering a bunch of quality and diversity metrics:
+- Input length
+- Output length
+- Reward score from a 1.4B Pythia model trained on open assistant data
+- Perplexity
+- MTLD: Measure of Textual Lexical Diversity [1]
```
Comment (Dahoas) [inline papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md:12]: Format quality/diversity metrics

File: papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md (hunk 1 above)
Comment (kcoost) [reply papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md]: Added to paper description

Comment (kcoost) [reply papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md]: Added to paper description

File: papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md (hunk 2 above)
Comment (kcoost) [reply papers/Instruction_Mining_When_Data_Mining_Meets_Large_Language_Model_Finetuning.md]: Added to paper description

**Pull request merged**